gp2gp-translator/src/main/resources/application.yml

```diff
@@ -9,7 +9,7 @@ spring:
   datasource:
     url: ${PS_DB_URL:jdbc:postgresql://localhost:5436}/patient_switching
     username: gp2gp_user
-    password: ${GP2GP_TRANSLATOR_USER_DB_PASSWORD}
+    password: ${GPC_FACADE_USER_DB_PASSWORD}
```

Comment:
This was previously GP2GP_TRANSLATOR_USER_DB_PASSWORD, did you mean to change it to facade?

Reply:
Good spot, updated.